frontend/src/index.tsx

```diff
@@ -15,4 +15,4 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+// serviceWorker.unregister();
```

Comment:
This is unnecessary, and causes a warning to be produced when compiling code. Why did you have to comment out this line?

Reply:
(By the way, it looks like you need to merge to `master`, there's a LOT of code that is left over from the previous pull request.)

Reply:
Yup, that's a really good point, thanks.

Reply:
I noticed that the `serviceWorker` was unnecessary, so I removed that line and the import.